core/context.go

```diff
@@ -26,17 +26,30 @@ import (
 	"go.uber.org/fx/service"
 )
 
-// Context embedds Host and go context for use
+// Context embeds Host and go context for use
```

Comment:
my point is that "go" should always be "Go' and that we should maybe say "Go stdlib context"